src/components/AudioTrack/Waveform.vue

```diff
@@ -203,6 +213,14 @@ export default {
     seek(event) {
       this.$emit('seeked', this.getPositionPercentage(event))
     },
+    seekJumpBackward() {
+      this.clearSeekProgress()
+      this.$emit('jumpSeekedBackward')
+    },
+    seekJumpForward() {
+      this.clearSeekProgress()
+      this.$emit('jumpSeekedForward')
+    },
```

Comment:
To maintain parity with mouse-based seek, maybe we could emit the same `seeked` event with the position percentage calculated in a different way?

Pseudocode:
```js
this.$emit('seeked', this.percentage + percentageDelta)
```